### PR TITLE
Add configuration ability

### PR DESCRIPTION
**Pull request by Valve**
This allows to set basic auth username + password and base_url in the configuration.
We've been using it in prod for about a week, all fine

### CODE REVIEW

File: lib/sixpack.rb [hunk 1]
```diff
@@ -103,9 +113,12 @@ def get_response(endpoint, params)
       query = Addressable::URI.form_encode(self.build_params(params))
 
       begin
-        res = http.start do |http|
-          http.get(uri.path + endpoint + "?" + query)
+        req = Net::HTTP::Get.new(uri.path + endpoint + "?" + query)
+        # basic auth
+        if @user && @password
```
Comment (zackkitzmiller) [inline lib/sixpack.rb]: Could you specifically document this?

Comment (Valve) [inline lib/sixpack.rb]: @zackkitzmiller done

**Comment by Valve** [comment]
Do you want me to squash the commits?

**Comment by zackkitzmiller** [comment]
Na, no need for that.

**Comment by Valve** [comment]
Thanks, Zack

**Pull request merged**